packages/media/src/createResource.js

```diff
@@ -99,6 +101,10 @@ import normalizeResourceSizes from './normalizeResourceSizes';
  * @property {string|null} lengthFormatted The formatted length for the "video" type.
  * @property {string|null} alt The user-readable accessibility label for the resource.
  * @property {boolean} local Whether the resource has been already uploaded to the server.
+ * @property {boolean} isOptimized Whether the resource has already been optimized.
+ * @property {boolean|null} isMuted Whether the resource has already been muted.
+ * @property {boolean|null} isExternal Whether the resource is externally hosted.
+ * @property {boolean|null} isPlaceholder Whether the resource is a placeholder.
```

Comment:
I don't think these are ever `null` because they have default values in the function signature below.

```suggestion
 * @property {boolean} isMuted Whether the resource has already been muted.
 * @property {boolean} isExternal Whether the resource is externally hosted.
 * @property {boolean} isPlaceholder Whether the resource is a placeholder.
```